euth/projects/templates/euth_projects/includes/project_tile.html

```diff
@@ -1,7 +1,13 @@
 {% load i18n thumbnail %}
-<div class="project-tile">
-    <a href="{% url 'project-detail' project.slug %}" class="project-tile-image-link"style="background-image: url('{{ project.image|thumbnail_url:'project_thumbnail' }}');">
-        <img src="{{ project.image|thumbnail_url:'project_thumbnail'}}" class="project-tile-image img-responsive"></a>
+<div class="project-tile {{ project.is_public|yesno:'public,private' }} {% if not project.image %} no-image {% endif %} {% if project.days_left <= 5 %} running-out {% endif %}">
+    <a href="{% url 'project-detail' project.slug %}" class="project-tile-image-link"
+       style="background-image: url('{% if project.image %} {{ project.image|thumbnail_url:'project_thumbnail' }} {% else %}/static/images/placeholder.png{% endif %}');">
```

Comment:
Loading files from static should be done via the`{% static '/images/placeholder.png'  %} tag. In order to use it you need to add `static` to `load`. More detail in the [django docs](https://docs.djangoproject.com/en/1.8/howto/static-files/).